tests/test_user_management.py

```diff
@@ -25,7 +31,27 @@ def test_success(self):
 
         self.assertCountEqual(actual.keys(), expected_keys)
 
-    def test_failure(self):
+    def test_user_by_token_failure(self):
         with self.assertRaises(requests.exceptions.HTTPError):
             _ = get_user_information('https://cloud-backend.lanalabs.com',
                                      token='not-a-valid-token ')
+
+    def test_list_users(self):
+        users = self.api.get_all_users()
+        self.assertGreaterEqual(len(users), 1)
+
+    def test_user_management(self):
+        resp_user_info = self.api.get_user_information_by_id('e44e8b55-1b28-4db7-9396-6acd4e94ee2a')
+
+        self.assertEqual(resp_user_info['role'], 'UserAdmin')
+
+        resp_user_creation = self.api.create_user('nonexisting@doesnotexistlana.org',
+                                                  'Analyst',
+                                                  resp_user_info['organizationId'],
+                                                  resp_user_info['backendInstanceId'])
+
+        self.assertEqual(resp_user_creation.status_code, 200)
+
+        resp_delete_user = self.api.delete_user(resp_user_creation.json()['id'])
```

Comment:
We might want to put the clean-up code into a class-method `tearDownClass`, so that a failing test doesn't leave a messy source system. Here a failure would not delete the user.

In this case we could add code that tries to delete all users that are created by our tests by giving the "ephemeral" users mail addresses that differ from the users for which we pass credentials.